e2e_tests/visual_regression_test.py

```diff
@@ -1,25 +1,29 @@
-import re
-
-from playwright.sync_api import expect
+import optuna
 from playwright.sync_api import Page
+import pytest
 
 
-def test_homepage_has_Playwright_in_title_and_get_started_link_linking_to_the_intro_page(
+@pytest.mark.parametrize("study_id", range(10))
```

Comment:
It is difficult to get study_size from fixtures, and spcified 10 here as magic number.